include/rppt_tensor_arithmetic_operations.h

```diff
@@ -70,8 +70,8 @@ RppStatus rppt_fused_multiply_add_scalar_host(RppPtr_t srcPtr, RpptGenericDescPt
  *          It multiplies each element of the source tensor by a corresponding element in the 'mulTensor',
  *          adds a corresponding element from the 'addTensor', and stores the result in the destination tensor.
  *          Support added for f32 -> f32 dataype.
- * \image html input150x150x4.gif Sample Input
- * \image html arithmetic_operations_fused_multiply_add_scalar_150x150x4.gif Sample Output
+ * \image html https://raw.githubusercontent.com/ROCm/rpp/develop/docs/data/doxygenInputs/input150x150x4.gif Sample Input
```

Comment:
I see you have changed this for arithmetic and geometric groups. 
There's 2 more under the "effects" group with GIFs - rppt_gaussian_noise_voxel_host() and rppt_gaussian_noise_voxel_hip()
Here -> https://github.com/ROCm/rpp/blob/develop/include/rppt_tensor_effects_augmentations.h#L512-L554

Reply:
Nice catch. Sorry I missed these.